tabular/setup.py

```diff
@@ -50,11 +50,14 @@
     'skex': [
         'scikit-learn-intelex<=2021.3',
     ],
+    'imodels': [
+        'imodels>=1.0.3',
+    ],
 }
 
 all_requires = []
 # TODO: Consider adding 'skex' to 'all'
-for extra_package in ['lightgbm', 'catboost', 'xgboost', 'fastai']:
+for extra_package in ['lightgbm', 'catboost', 'xgboost', 'fastai', 'imodels']:
```

Comment:
Since `imodels` will probably not be a by-default installed dependency, we will need to change this logic prior to merging PR, because `autogluon` installs `autogluon.tabular[all]` during `pip install autogluon`. We may want to change it to install `autogluon.tabular[core]` (with core being the current `all`) and add `imodels` in `all` or require `imodels` to be installed separately from `all`.

Reply:
CI can still use `imodels` so long as we install `imodels` in the CI run.

Reply:
switched over!